config/config_test.go

```diff
@@ -88,20 +79,14 @@ func testValidateTesthelper(
 		cfgService.Attributes["map_rate_sec"] = -1
 		_, err = newConfig(cfgService)
 
-		if !cloudStoryEnabled {
-			test.That(t, err, test.ShouldBeError, newError("cannot specify map_rate_sec less than zero"))
-		} else {
-			test.That(t, err, test.ShouldBeNil)
-		}
```

Comment:
test revert to reflect config validation revert so that we can make config changes without breaking cartographer